python/cli.schema.json

```diff
@@ -57,6 +57,22 @@
         }
       ]
     },
+    "complete_text": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": [
+        {
+          "$ref": "#/definitions/TextCompletionRequest"
+        },
+        {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RewriteChoice"
```

Comment:
Is there a particular reason? Would that make something on the front-end easier? I only ask because `complete_text` follows an almost identical code path to `rewrite` and returns a type that is structured exactly the same.

Reply:
@sehyod I'm good with that. Maybe we make it more specific with `TextSuggestionChoice`?

Reply:
I think that makes sense to have a new return type bc is a different operation, even though internally is sharing code and type definitions.

If both operations (rewrite and completion) are a almost identical, we can maybe rewrite both as a `suggestion` command with a `type` parameter to specify if we want a "rewrite" or a "completion".

As I think that both (rewrite and completion) can evolve differently I think that makes sense to have them as different commands with different request and response types.

Reply:
I concede and will update this to return a `TextCompletionChoice`